app/views/documents/_secondary_navigation.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= render "components/secondary_navigation", {
+  aria_label: "Document",
+  items: [
+    {
+      label: t("documents.secondary_navigation.summary"),
+      href: document_path,
+      current: current_page?(document_path),
+      data_attributes: {
+        gtm: "summary-tab"
+      }
+    },
+    {
+      label: t("documents.secondary_navigation.history"),
+      href: history_path_url,
+      current: current_page?(history_path_url),
```

Comment:
The use of history_path_url seems a mistake here. As discussed this seems to be because we named the route history_path rather than just history.